conf/vagrant/provisioning/roles/https/templates/san.cnf

```diff
@@ -13,7 +13,6 @@ countryName      = US
 # The extentions to add to a self-signed cert
 subjectKeyIdentifier = hash
 basicConstraints     = critical,CA:false
-subjectAltName       = DNS:{{ hostname }},DNS:www.{{ hostname }}{% for host in extra_hostnames  %}
-,DNS:{{ host }}{% endfor %}
+subjectAltName       = DNS:{{ hostname }},DNS:www.{{ hostname }}{% for host in extra_hostnames  %},DNS:{{ host }}{% endfor %}
 
```

Comment:
This new line is actually necessary (as I assume you've already gleaned). Without it, the two lines don't end up with a linebreak:
```